src/main/native/cpp/view/AnalyzerPlot.cpp

```diff
@@ -278,4 +306,22 @@ void AnalyzerPlot::DisplayTimeDomainPlots() {
       }
     }
   }
+  if (m_fitNextPlot[6]) {
+    ImPlot::FitNextPlotAxes();
+  }
```

Comment:
We should clamp the y minimum to 0 since negative periods are impossible.

Reply:
Do I set the plot limits or should I remove any negative datapoints?